### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by igorcandeia**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -57,23 +59,108 @@ Travis CI and AppVeyor are both continuous integration services configured to be
...
+4.2 Activating Appveyor:
+
+1. Open https://www.appveyor.com/ and sign in with github account
+2. On the dashboard click on `New Project`
+3. Wait (no kidding, takes a time to load all projects).
+4. Search for the <project_name> and click on `Add`
+
+5. Setup deploy to PyPI
+
+Make sure to have an account on `PyPI`_
```
Comment (igortg) [inline README.rst]: Again? You already warned about that in right above

File: README.rst [hunk 2]
```diff
@@ -57,23 +59,108 @@ Travis CI and AppVeyor are both continuous integration services configured to be
...
+Make sure to have an account on `PyPI`_ 
+
+Travis CI will be also used to deploy the application to `PyPI`_, for more details check the section ``Release`` on the README file from your project.
+In order to set up this configuration, it's necessary to encrypt you `PyPI`_ password on ``.travis.yml``
+
+Run the commands below on the root directory of your project.
 
 .. code-block:: bash
 
       1) travis encrypt --add deploy.password
```
Comment (igortg) [inline README.rst:91]: Shouldn't the user install `travis` CLI first?

File: README.rst [hunk 3]
```diff
@@ -57,23 +59,108 @@ Travis CI and AppVeyor are both continuous integration services configured to be
...
+    only:
+    - master
+    - "/v(\\d+\\.)*\\d/"
+
+
+6. Set Up ReadTheDocs
+
+`ReadTheDocs`_ hosts documentation for the open source community.
 
+1. Log into your account at `ReadTheDocs`_ . If you don't have one, create one and log into it.
```
Comment (igortg) [inline README.rst]: Leave just _Log in to your ReadTheDocs account_ on this line. The rest is pretty obvious

File: README.rst [hunk 4]
```diff
@@ -57,23 +59,108 @@ Travis CI and AppVeyor are both continuous integration services configured to be
...
 
-Travis CI will be also used to deploy the application to `PyPI`_, for more details check the section `Release` on the README file from your project.
-In order to set up this configuration, it's necessary to encrypt you PyPi password on `.travis.yml`
+4.1 Activating Travis:
+
+1. Open https://travis-ci.org/ sign in with github account
+2. Access https://travis-ci.org/ESSS/<package_name>
+3. Click on Activate repository
+
+4.2 Activating Appveyor:
```
Comment (nicoddemus) [inline README.rst]: It is `AppVeyor` ;)

File: README.rst [hunk 5]
```diff
@@ -57,23 +59,108 @@ Travis CI and AppVeyor are both continuous integration services configured to be
...
+  deploy:
+    password:
+      secure: <encrypted password>
+
+
+Copy the ``<encrypted password>`` and replace the ``REPLACE`` text with the encrypted password.
+
+Do not forget to delete the deploy and password newly created tags
+
+.. code-block:: python
```
Comment (nicoddemus) [inline README.rst]: `.. code-block:: yaml`

File: README.rst [hunk 6]
```diff
@@ -57,23 +59,108 @@ Travis CI and AppVeyor are both continuous integration services configured to be
...
+      
+      On Linux:
+         3) Press CTRL+D
+      Windows:
+         3) Press Enter
+         4) On the new line Press CTRL+Z
+
+The command line from ``travis`` will create a new entry called "Deploy" on your ``.travis.yml``  file. 
+
+.. code-block:: python
```
Comment (nicoddemus) [inline README.rst]: `.. code-block:: yaml`

File: README.rst [hunk 7]
```diff
@@ -57,23 +59,108 @@ Travis CI and AppVeyor are both continuous integration services configured to be
...
+          secure: <encrypted password>
+        on:
+          tags: true
+          repo: ESSS/<package name>
+          python: 3.6
+  branches:
+    only:
+    - master
+    - "/v(\\d+\\.)*\\d/"
+
```
Comment (nicoddemus) [inline README.rst:143]: We should also mention that after the first release (made by pushing a tag), the project will appear on PyPI and the user should add more maintainers to it. 👍

**Review by nicoddemus — CHANGES_REQUESTED** [review]
Overall good work!

**Review by williamjamir — APPROVED** [review]

**Review by nicoddemus — APPROVED** [review]

**Pull request merged**